ingest/build.sbt

```diff
@@ -24,13 +24,13 @@ assemblyMergeStrategy in assembly := {
   case _ => MergeStrategy.first
```

Comment:
mb [this](https://github.com/pomadchin/geotrellis-chatta-demo/blob/168cd59f4c066aa541b199a33e1bf4f21e5581b0/geotrellis/build.sbt#L37-L42) way to define assembly strategy is more compact?